pkg/webhook/controlplane/genericmutator/mutator.go

```diff
@@ -66,6 +68,9 @@ type Ensurer interface {
 	EnsureAdditionalFiles(context.Context, *[]extensionsv1alpha1.File) error
 }
 
+// TODO: use 'v1alpha1constants.OperatingSystemConfigUnitNameGardenerUserService' when new gardener version is released
```

Comment:
Please do this already now instead of redeclaring the constant here (as the PR is merged already). Vendor the master branch.